cmake/amr-wind-utils.cmake

```diff
@@ -71,6 +71,15 @@ macro(init_amrex)
   endif()
 endmacro(init_amrex)
 
+macro(init_amrex_hydro)
+  set(AMREX_HYDRO_SUBMOD_LOCATION "${CMAKE_SOURCE_DIR}/submods/AMReX-Hydro")
+  include(${CMAKE_SOURCE_DIR}/cmake/set_amrex_hydro_options.cmake)
+  if (AMR_WIND_ENABLE_CUDA)
+    set(HYDRO_GPU_BACKEND CUDA CACHE INTERNAL "")
```

Comment:
I think this can be removed since it happens in the options just before it.

Reply:
@jrood-nrel Good catch! I have removed this here - https://github.com/Exawind/amr-wind/pull/551/commits/dd621eb635d727fe286ba8e5b8cb8bb6d5a624fe